recipes/minimize_access.rb

```diff
@@ -53,3 +53,9 @@
   mode '0750'
   not_if { node['os-hardening']['security']['users']['allow'].include?('change_user') }
 end
+
+# /var/log should restricted to root or syslog on ubuntu > 14.04
+directory '/var/log' do
+  owner 'root'a
```

Comment:
@shoekstra could you please fix this? (`a`) ;-)

Reply:
Whoops, fix incoming.  Good spot :)